src/main/scala/chisel3/util/experimental/BoringUtils.scala

```diff
@@ -128,6 +128,7 @@ object BoringUtils {
     * @return the name used
     * @note if a uniqueName is not specified, the returned name may differ from the user-provided name
     */
+  @deprecated("Please use the new Boring API instead (BoringUtils.bore(source)). This will be removed in Chisel 7.0", "Chisel 7.0")
```

Comment:
```suggestion
  @deprecated("Please use the new Boring API instead (BoringUtils.bore(source)). This will be removed in Chisel 7.0", "Chisel 6.0")
```
This 2nd argument is the "since" argument so should be Chisel 6